src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -249,6 +296,10 @@ class ExtentPlacementManager {
     return background_process.run_until_halt();
   }
 
+  std::vector<RandomBlockManager*> get_rb_managers() {
+    return background_process.get_rb_group()->get_rb_managers();
+  }
```

Comment:
A device tier with its Cleaner needs to handle the same kind of devices on its own, and EPM is managing tiers in general.

I think it's not reasonable to expose rb_managers from EPM.